AppBundles/ExportDrawingAsPdfPlugin/Automation.cs

```diff
@@ -37,64 +37,27 @@ public override void ExecWithArguments(Document doc, NameValueMap map)
             using (new HeartBeat())
             {
                 var dir = System.IO.Directory.GetCurrentDirectory();
-                var drawingToGenerate = map.Count>1 ? map.Item["_2"] : null;
+                var drawingToGenerate = map.Item["_2"];
 
                 if (drawingToGenerate == null)
                 {
                     LogTrace("Drawing not specified !");
-                    return;
                 }
 
                 LogTrace("Drawing to generate PDF: {0}", drawingToGenerate);
 
-                string drawing = null;
-                if (drawingToGenerate == null)
```

Comment:
this was search for drawing with the same name as starting assembly, do we want to skip this behavior?

Reply:
It is not used anymore if I understand that correctly. If so, lets remove it, it is only misleading why it is there.